relay-config/src/config.rs

```diff
@@ -550,6 +579,7 @@ impl Default for Http {
             host_header: None,
             auth_interval: Some(600), // 10 minutes
             auth_grace_period: 10,
+            encoding: HttpEncoding::Gzip,
```

Comment:
Is this PR sneaking in an unrelated change (adding GZip support for upstream request body) or is it me?

Reply:
You're right, pushed a commit to the wrong branch.